MAINTAINERS.md

```diff
@@ -5,3 +5,4 @@ TJBot was lovingly created by researchers at IBM Research.
 - [Maryam Ashoori](https://github.com/maryamashoori) is the creator of TJBot and is the ringleader of our merry band of cardboard robots.
 - [Justin Weisz](https://github.com/jweisz) created the TJBot library and generally tries to bring order to our chaos.
 - [Victor Dibia](https://github.com/victordibia) created TJBot’s initial receipes and is our master of demos.
+- [Bruno Braga](https://github.com/bgbraga) created TJBot bootstrap and helping to maintain the project.
```

Comment:
I tend to agree with this approach. Its what we had until recently. Will work on a PR for this to possibly reflect a core team and collaborators as the project grows.